tests/test_stability_funs.py

```diff
@@ -596,6 +596,8 @@ def test_ballooning_stability_eval():
         # different numerics than "ideal ballooning lambda" so that we can verify them
         # against one another
         psi_b = data01["Psi"][-1] / (2 * jnp.pi)
+        # Calculating a_N accurately requires a QuadratureGrid
+        # but for some reason it gives the correct value even with a LinearGrid here.
```

Comment:
`eq.compute` will automatically override the grid to the correct value in case the user forgets, which is why it gives the correct value even though you give it a lineargrid.